### DISCUSSION TITLE
Is there a way that could connect supported datasource through SSH Tunnel

### DISCUSSION THREAD

**Discussion by TonyEiven** [diesel-rs/diesel#3220] [Q&A] [answered]
I'm a beginner on `diesel`, recently I'm on an assessment with Rust ORM, and come across the function of establishing a connection through SSH Tunnel seems likely not feasible in diesel, is there a way or workaround that could connect datasource through SSH Tunnel?

**Comment by weiznich** [comment, marked as answer]
I'm not aware of any database system that supports such an setup directly. You might want to use some transparent proxy setup for such things.
